CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ set( CMAKE_CXX_STANDARD_REQUIRED ON )
 set( CMAKE_CXX_EXTENSIONS OFF )
 
 set( SFGUI_MAJOR_VERSION 0 )
-set( SFGUI_MINOR_VERSION 3 )
-set( SFGUI_REVISION_VERSION 2 )
+set( SFGUI_MINOR_VERSION 4 )
+set( SFGUI_REVISION_VERSION 0 )
```

Comment:
Why does adding some build scripts call for a major version increase?

Reply:
Sorry, I should have commented on that. The latest release is [`0.4.0`](https://github.com/TankOs/SFGUI/releases/tag/0.4.0), the CMake version was not updated. I thought about shoe-horning this change in here since I already touched the configuration. I can remove that and open a different PR with this edit inside.

Reply:
Oh, I didn't realize that Tank tagged it but didn't update the version.
Yeah, usually best to keep different changes separated, so one doesn't need to wait for the other to be merged and it's easier to review. 🙂

Reply:
Done in #75, I'll remove the fix on this branch.